tensorflow_graphics/projects/mesh_rcnn/structures/mesh.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2020 The TensorFlow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data Structure for Meshes"""
+
+import tensorflow as tf
+
+from tensorflow_graphics.geometry.convolution import utils
+from tensorflow_graphics.projects.mesh_rcnn.util import padding
+
+
+class Meshes:
+  """Wrapper for batching of Meshes with unequal size."""
```

Comment:
batching of Triangle Meshes

Q: It should be feasible to generalize this to non triangular face meshes, in D-dim space (not 3D). You dont need to do it in this PR, but good to know?

Reply:
I think it should be possible with very little effort. Overall, I would appreciate your feedback on this class and the interfaces/functionalities it provides (also see the more evolved version in #441). Is this something that you can imagine using as a main interface to TFG ops working with meshes? I would love to work on this even after GSoC is finished.